packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -2735,6 +2839,53 @@ export class Vector3 {
         Quaternion.RotationQuaternionFromAxisToRef(axis1, axis2, axis3, quat);
         quat.toEulerAnglesToRef(ref);
     }
+ 
+	/**
+	 * Gets the rotation on all axes between two vectors
+	 * @param vector0 defines the first vector
+	 * @param vector1 defines the second vector
+	 * @param offset defined the amount to offset the output rotation
+	 * @returns a new Vector3 in the form (pitch, yaw, roll)
+	 * Output roll without offset will always be 0
+	 */
+	public static RotationBetweenVectors(vector0: DeepImmutable<Vector3>, vector1: DeepImmutable<Vector3>, offset: DeepImmutable<Vector3>): Vector3{

```

Comment:
RotationBetweenVectors here is a static but called on line 1725 as a Vector3 method.